[objc] Make objcguide.xml link to objcguide.md 🔄

stephanemoore opened the pull request:
There are links out in the wild to the old XML style guide. An XML file that links to the new style guide location ensures that those links are not completely broken.

stephanemoore committed:
[objc] Make objcguide.xml link to objcguide.md 🔄

There are links out in the wild to the old XML style guide. An XML file that links to the new style guide location ensures that those links are not completely broken.

markmentovai commented:
This will be served at https://google.github.io/styleguide/objcguide.xml, so the relative link will point to https://google.github.io/styleguide/objcguide.md, which probably isn’t what you want (it’s raw, not rendered). You probably want https://github.com/google/styleguide/blob/gh-pages/objcguide.md.

stephanemoore committed:
[objc] Fix the link to the new Objective-C style guide ⛓

The link can't be relative because the new style guide is served as markdown and the endpoint serving the XML file serves raw content.

stephanemoore commented:
Good catch 😅 I updated the link to https://github.com/google/styleguide/blob/gh-pages/objcguide.md.

stephanemoore commented:
Thanks for the review!
